operator/cmd/root.go

```diff
@@ -98,11 +98,12 @@ var (
 		// Runs the gops agent, a tool to diagnose Go processes.
 		gops.Cell(defaults.GopsPortOperator),
 
-		// Provides Clientset, API for accessing Kubernetes objects.
-		k8sClient.Cell,
```

Comment:
Suggestion: make the `k8sclient.Cell` and `ClientBuilderCell` take a `ClientParams` object, which is provided by an Agent or Operator cell.

Reply:
Looks like that's just what you did, looks nice :-)